bcda/models/SwaggerStructs.go

```diff
@@ -143,7 +143,7 @@ type ResourceTypeParam struct {
 
 // swagger:parameters bulkPatientRequest bulkGroupRequest
 type SinceParam struct {
-        // (Optional) Only include resource versions that were created at or after the given instant in time.  Format of string must align with the FHIR dateTime datatype.
+        // (Optional) Only include resource versions that were created at or after the given instant in time.  Format of string must align with the FHIR dateTime datatype.  Example: `2020-02-13T08:00:00.000-05:00`
```

Comment:
Can this be a Swagger example?
- [Spec](https://swagger.io/docs/specification/adding-examples/)
- [SSAS precedence](https://github.com/CMSgov/bcda-ssas-app/blob/c404741a27b48667768c24166d86b1ff020cdf75/ssas/swagger_structs.go#L183) (though admittedly not a param example)

Reply:
The `example` annotation is not working in this repo.  I wonder if its set up differently in `bcda-ssas-app`?

Reply:
Looks like it's simply different for parameters.